pom.xml

```diff
@@ -162,8 +160,8 @@
       <scope>test</scope>
       <exclusions>
         <exclusion>
-          <groupId>org.springframework.boot</groupId>
-          <artifactId>spring-boot-starter-logging</artifactId>
+          <groupId>org.junit.vintage</groupId>
```

Comment:
Vintage no longer needs to be excluded.
The latest spring boot starter test does not include it.
Please remove the exclusion.